src/integrationTest/java/uk/gov/hmcts/darts/annotation/AnnotationGetTest.java

```diff
@@ -131,14 +132,13 @@ private AnnotationDocumentEntity createValidAnnotationDocumentForDownload(UserAc
     }
 
     private AnnotationEntity someAnnotationCreatedBy(UserAccountEntity userAccount) {
-        var annotation = minimalAnnotationEntity();
-        annotation.setDeleted(false);
-        annotation.setCurrentOwner(userAccount);
-        annotation.setCreatedBy(userAccount);
-        annotation.setLastModifiedBy(userAccount);
-        annotation.addHearing(someMinimalHearing());
-        dartsPersistence.save(annotation);
-        return annotation;
+        var annotation = PersistableFactory.getAnnotationTestData().someMinimal().getBuilder()
```

Comment:
Would be better to have someMinimalBuilder() instead of .someMinimal().getBuilder()

then have .someMinimal() {
return someMinimalBuilder().build()
}